tasks/eva_3618/findAndDeleteSplitOperations.py

```diff
@@ -0,0 +1,104 @@
+        for collection in CVE_OPS_COLLECTIONS:
+            cve_coll = mongo_conn['eva_accession_sharded'][collection]
+            cve_query = {
+                'inactiveObjects.asm': {'$in': asm_list},
+                'eventType': 'RS_SPLIT',
+                '$expr': {'$eq': ['$accession', '$splitInto']}
+            }
+            cursor = cve_coll.find(cve_query, no_cursor_timeout=True)
+
+            for cve_ops_doc in cursor:
+                cve_docs.append(cve_ops_doc)
+                cve_ops_ids.add(cve_ops_doc['_id'])
+                cve_ops_accs.add(cve_ops_doc['accession'])
+
+        logger.info(f'Number of CVE documents: {len(cve_docs)}')
+        # Save CVE documents to file
+        cve_docs_file = os.path.join(working_dir, "cve_docs.json")
+        with open(cve_docs_file, 'w') as f:
+            json.dump(cve_docs, f, default=str)
+
+        # get sve ops ids
+        for collection in SVE_OPS_COLLECTIONS:
+            sve_coll = mongo_conn['eva_accession_sharded'][collection]
+            sve_query = {
+                'inactiveObjects.rs': {'$in': list(cve_ops_accs)}
+            }
+            cursor = sve_coll.find(sve_query, no_cursor_timeout=True)
+
+            for sve_ops_doc in cursor:
+                rs = f"rs{sve_ops_doc['inactiveObjects'][0]['rs']}"
+                reason = f'SS was associated with the split RS {rs} that was split from {rs} after remapping.'
+                if sve_ops_doc['reason'] == reason:
+                    sve_docs.append(sve_ops_doc)
+                    sve_ops_ids.add(sve_ops_doc['_id'])
+
+        logger.info(f'Number of SVE documents: {len(sve_docs)}')
+        # Save SVE documents to file
+        sve_docs_file = os.path.join(working_dir, "sve_docs.json")
+        with open(sve_docs_file, 'w') as f:
+            json.dump(sve_docs, f, default=str)
+
+        # delete sve operations
+        for collection in SVE_OPS_COLLECTIONS:
+            sve_coll = mongo_conn['eva_accession_sharded'][collection]
+            sve_del_query = {'_id': {'$in': list(sve_ops_ids)}}
+            result = sve_coll.delete_many(sve_del_query)
+            logger.info(f'Deleted {result.deleted_count} SVE documents from {collection}')
+
+        # delete cve operations
+        for collection in CVE_OPS_COLLECTIONS:
+            cve_coll = mongo_conn['eva_accession_sharded'][collection]
+            cve_del_query = {'_id': {'$in': list(cve_ops_ids)}}
+            result = cve_coll.delete_many(cve_del_query)
+            logger.info(f'Deleted {result.deleted_count} CVE documents from {collection}')
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Find and delete RS_SPLIT operations that were split into the same RS',
+                                     add_help=False)
+    parser.add_argument("--private-config-xml-file", help="ex: /path/to/eva-maven-settings.xml", required=True)
+    parser.add_argument("--working-dir", help="ex: /path/to/dir where ", required=True)
```

Comment:
```suggestion
    parser.add_argument("--working-dir", help="ex: /path/to/dir where deleted documents will be logged", required=True)
```
(or something similar)

Reply:
updated